src/pages/dar_application/CollaboratorList.js

```diff
@@ -0,0 +1,228 @@
+            defaultValue: collaborator.name || '',
+            className: 'form-control',
+            required: true,
+            onBlur: (e) => updateAttribute(index, 'name', e.target.value)
+          })
+        ]),
+
+        div({ className: 'col-lg-6 col-md-6 col-sm-6 col-xs-12' }, [
+          label({ className: 'control-label' }, [`${collaboratorLabel} Title*`]),
+          input({
+            type: 'text',
+            name: `collaborator-${collaborator.uuid}-title`,
+            defaultValue: collaborator.title || '',
+            className: 'form-control',
+            required: true,
+            onBlur: (e) => updateAttribute(index, 'title', e.target.value)
+          })
+        ])
+      ]),
+      div({ className: 'row'}, [
+        div({ className: 'col-lg-6 col-md-6 col-sm-6 col-xs-12' }, [
+          label({ className: 'control-label' }, [`${collaboratorLabel} NIH eRA Commons ID*`]),
+          input({
+            type: 'text',
+            name: `collaborator-${collaborator.uuid}-eraCommonsID`,
+            defaultValue: collaborator.eraCommonsId || '',
+            className: 'form-control',
+            required: true,
+            onBlur: (e) => updateAttribute(index, 'eraCommonsId', e.target.value)
+          })
+        ]),
+        div({
+          className: 'col-lg-6 col-md-6 col-sm-6 col-xs-12'
+        }, [
+          label({
+            className: 'control-label'
+          }, [`${collaboratorLabel} Email`]),
+          input({
+            type: 'email',
+            name: `collaborator-${collaborator.uuid}-email`,
+            defaultValue: collaborator.email || '',
+            className: 'form-control',
+            required: true,
+            onBlur: (e) => updateAttribute(index, 'email', e.target.value)
+          })
+        ])
+      ]),
+      div({ className: 'row', isRendered: showApproval }, [
+        div({className: 'col-lg-12 col-md-12 col-sm-12 col-xs-12'}, [
+          p({ className: 'control-label rp-choice-questions'},[
+            `Are you requesting permission for this member of the Internal Lab Staff to be given "Designated Download/Approval" status? This
+            indication should be limited to individuals who the PI designates to download data and/or share the requested data with other Internal
+            Lab Staff (i.e., staff members and trainees under the direct supervision of the PI)`
+          ]),
+          //mapped output of yes and no radio buttons for the list
+          //YesNoRadioGroup is not used because it assumes only one instance of a question exists at a time
+          //this component requires the question to be rendered multiple times, therefore index needs to be passed to id and key
+          //YesNoRadioGroup can probably be refactored to handle this, the issue would be rewritting other uses of the component to match the updated form
+          //Thus the YesNoRadioGroup refactor should be handled on a separate PR
```

Comment:
Agreed on these comments ... I'm really not a fan of that component. In `VoteQuestion`, I essentially wrote my own version that's less bootstrappy and doesn't rely on the strange layered effect in `YesNoRadioGroup`. I recommend deprecating the old component and moving forward with the pattern you have here.

Reply:
Not for this PR, but a follow on to the YNRG conversation. If you look closely, the selection of the selected radio buttons are not perfectly centered. 
![Screen Shot 2020-08-25 at 2 20 09 PM](https://user-images.githubusercontent.com/116679/91212484-4a2fdb80-e6de-11ea-8217-495f728151fa.png)